stable/k8s-event-logger/values.yaml

```
@@ -1,4 +1,5 @@
 image:
+  containerName: k8s-event-logger
```

It's not part of the image so I would not put it under here:

```suggestion
containerName: k8s-event-logger

image:
```